### PR TITLE
[KOGITO-2408] Spring Kafka default property should be `spring.kafka.bootstrap-servers`

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-2408

Removing old behavior of `kafka.bootstrap.servers`.
We should use default behavior of the runtime (here Spring Boot) to avoid confusion on setup.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Review by Sgitario — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Comment by radtriste** [comment]
jenkins retest this

File: process-kafka-quickstart-springboot/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,4 +1,12 @@
-kafka.bootstrap.servers=localhost:9092
+spring.kafka.bootstrap-servers=localhost:9092
```
Comment (cristianonicolai) [inline process-kafka-quickstart-springboot/src/main/resources/application.properties:1]: @radtriste the JIRA says to use _kafka.bootstrap.servers_ and in here you are moving to _spring.kafka.bootstrap-servers_ does the JIRA needs update?

**Review by cristianonicolai — COMMENTED** [review]
@radtriste  @ricardozanini does this impacts the operator?

File: process-kafka-quickstart-springboot/src/main/resources/application.properties (hunk 1 above)
Comment (radtriste) [reply process-kafka-quickstart-springboot/src/main/resources/application.properties:1]: sorry forgot to update JIRA description ...
we want to use the default spring config property name.
Operator is already configured with `spring.kafka.bootstrap-servers`, that is why the tests were failing. Witht that update it should be fine

**Review by cristianonicolai — APPROVED** [review]

**Comment by radtriste** [comment]
@cristianonicolai job is green :D

**Pull request merged**